airbyte-integrations/connectors/source-gitlab/source_gitlab/source.py

```diff
@@ -40,6 +44,34 @@
 )
 
 
+class GitlabOauth2Authenticator(SingleUseRefreshTokenOauth2Authenticator):
+    def __init__(self, connector_config, *args, **kwargs):
+        super().__init__(connector_config, *args, **kwargs)
+        access_token = connector_config["credentials"].get("access_token")
+        if not access_token:
+            return
+        api_url = self._connector_config["api_url"]
+        token_valid, access_token_info = self.get_access_token_info(api_url, access_token)
+        if not token_valid:
+            return
+        now = pendulum.now()
+        self.access_token = access_token
+        self.set_token_expiry_date(now.add(seconds=access_token_info["expires_in"]))
```

Comment:
```suggestion
        self.set_token_expiry_date(now, access_token_info["expires_in"])
```
this function expects two arguments: `initial_time: pendulum.DateTime, value: Union[str, int]`